pkg/cli/session/session.go

```diff
@@ -178,69 +176,14 @@ func (e *BackplaneSession) Start() error {
 	cmd.Dir = e.Path
 	err = cmd.Run()
 	if err != nil {
-		return fmt.Errorf("error while running cmd. error %v", err)
-	}
-
-	err = e.killChildren()
-	if err != nil {
-		return err
+		fmt.Println("error while running cmd.")
```

Comment:
we do not need to return if there is error?

Reply:
Got it and return the error